includes/class-sensei-grading.php

```diff
@@ -823,16 +823,18 @@ private function deprecated_get_redirect_url() {
 	}
 
 	public function add_grading_notices() {
-		if ( isset( $_GET['page'] ) && $this->page_slug == $_GET['page'] && isset( $_GET['message'] ) && $_GET['message'] ) {
-			if ( 'graded' == $_GET['message'] ) {
-				$msg = array(
-					'updated',
-					__( 'Quiz Graded Successfully!', 'sensei-lms' ),
-				);
-			}
+		$page    = isset( $_GET['page'] ) ? sanitize_text_field( wp_unslash( $_GET['page'] ) ) : false;
```

Comment:
This method came up in my possibly unset variable inspector. I refactored it a bit.